ma/CMakeLists.txt

```diff
@@ -56,6 +56,8 @@ set(HEADERS
   ma.h
   maInput.h
   maMesh.h
+  maMap.h
```

Comment:
@avinmoharana 

Are these two need to be here, or is this done for debugging? If this is done for debugging, please revert this back.